app/graphql/types/user_recipe_type.rb

```diff
@@ -1,9 +1,11 @@
 module Types
   class UserRecipeType < Types::BaseObject
     field :id, ID, null: false
```

Comment:
Didn't know you could have type of ID, instead of using Integer. Any benefits of using this over Integer?